tensorflow/tools/ci_build/windows/libtensorflow_cpu.sh

```diff
@@ -76,14 +76,12 @@ zip -j ${DIR}/libtensorflow_jni-cpu-windows-$(uname -m).zip \
 rm -f ${DIR}/tensorflow_jni.dll
 
 # Zip up the .dll, LICENSE and include files for the C library.
+mkdir -p ${DIR}/include/tensorflow/c
+mkdir -p ${DIR}/lib
+cp bazel-bin/tensorflow/libtensorflow.so ${DIR}/lib/tensorflow.dll
```

Comment:
I think it's fine. Until the time we create the .Zip for the C library, we're in the root of the git repo, so at that time bazel-bin is fine. Or did I miss something?